monitor/census.go

```diff
@@ -1987,6 +1996,10 @@ func AIResultDownloaded(ctx context.Context, pipeline string, model string, down
 	}
 }
 
+func AIFirstSegmentDelay(delay int64) {
+	stats.Record(census.ctx, census.mAIFirstSegmentDelay.M(delay))
```

Comment:
I wonder if we may want to tag it with Orchestrator's address? 

I can imagine that for the given Gateway, only some Os may be slow.

Reply:
Yes. That would be interesting information.

The question is how to get Orchestrator info in `startTrickleSubscribe()` function.
Where of if does it live on the Gateway code? It must has `OrchestratorInfo` struct form I think

Reply:
You can pass [host](https://github.com/livepeer/go-livepeer/blob/c8cbe797a08ab91bf5b116e04dd55731231e91e2/server/ai_process.go#L1060) to `startTrickleSubscribe()`.

This `host` is good as a tag. It's the IP/DNS of the Orchestrator.

Reply:
OK. I took entire orchInfo and unified it a bit on the census-side